verify-node-version.ts

```diff
@@ -4,10 +4,20 @@
 "use strict";
 import {EOL} from "os";
 
-export function verifyNodeVersion(version: string): void {
+// These versions cannot be used with CLI due to bugs in the node itself.
+// We are absolutely sure we cannot work with them, so inform the user if he is trying to use any of them and exit the process.
+let versionsCausingFailure = ["0.10.34", "4.0.0", "4.2.0", "5.0.0"];
+
+export function verifyNodeVersion(supportedVersionsRange: string): void {
 	require("colors");
 	let nodeVer = process.version.substr(1);
-	let checkSatisfied = require("semver").satisfies(nodeVer, version);
+
+	if (versionsCausingFailure.indexOf(nodeVer) !== -1) {
+		console.error(`${EOL}The installed version of node.js '${nodeVer}' is not supported. Please use any version in the following range: ${supportedVersionsRange}.${EOL}`.red.bold);
```

Comment:
`${EOL}Node.js '${nodeVer}' is not supported. To be able to work with the Telerik AppBuilder CLI, install any Node.js version in the following range: ${supportedVersionsRange}.${EOL}`.red.bold